### PR TITLE
Andrewrutter/kkb 28 get events backend

### PR DESCRIPTION
**Pull request by andlrutt**
Very similar to PR#13, [KKB-33-get-event](https://github.com/hack4impact-utk/keep-knox-beautiful/tree/kemalfidan/KKB-33-get-event), except it returns ALL events in the database. Code is almost the same, but the `getEvents()` function uses `.find({})` instead of `.findById(id)` to return all the events in the DB in an array, which will eventually be paginated. 

Below is testing using postman and a web browser on dummy data in the DB. Actual DB contents in compass are on the right. Also added unit tests.
postman
localhost

### CODE REVIEW

**Review by rossketron — APPROVED** [review]
LGTM!

File: server/actions/Event.ts [hunk 1]
```diff
@@ -20,6 +20,20 @@ export const getEvent = async function (id: string) {
...
 };
 
+/**
+ * @returns All events.
+ */
+export const getEvents = async function () {
+    await mongoDB();
+
+    const events = (await EventSchema.find({})) as Array<Event>;
+    if (events == null) {
```
Comment (kemalfidan) [inline server/actions/Event.ts]: Can you change this to `if (!events || !events.length)`? `!events` is the same thing as you have right now. But I think we'd also need a check the array length here.

File: tst/server/Event.test.ts [hunk 2]
```diff
@@ -32,6 +32,24 @@ describe("getEvent() tests", () => {
     });
 });
 
+describe("getEvents() tests", () => {
```
Comment (kemalfidan) [inline tst/server/Event.test.ts:35]: Good tests 👍

File: server/actions/Event.ts (hunk 1 above)
Comment (andlrutt) [reply server/actions/Event.ts]: Done!

**Review by kemalfidan — APPROVED** [review]

**Pull request merged**